app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,61 @@
+class ListsController < ApplicationController
+  before_action :require_signed_in!
+
+  def index
+    @lists = List.all
+  end
+
+  def show
+    @list = List.includes(:tasks).find(params[:id])
+    if !check_owner(@list)
+      redirect_to root_path
+    end
+    redirect_to list_tasks_url(@list)
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def create
+    @list = List.new(list_params)
+    @list.user_id = current_user.id
```

Comment:
Hello, I have done these on two steps to make a distinction between using the `params` permitted in the private methods and the assigning a user_id. Another reason is that I do this out of habit, since as a project grows many times the required attributes for a model/table also grow and it's easy to just throw a debugger between the two to check if the right `params` are being passed.

Reply:
I also forgot to mention, I believe that adding the extra field of user_id ie: `List.new(list_params, user_id: current_user.id)` creates an argument error since it gets an options hash plus the additional `user_id` argument.